index.js

```diff
@@ -119,16 +100,81 @@
   //  $$type :: String
   var $$type = '@@type';
 
-  //  type :: Any -> String
+  //  RPARSE :: RegExp
+  var RPARSE = /^([^]+)\/([^]+?)(?:@(\d+))?$/;
```

Comment:
I would like to avoid the `\/` in the pattern. I'm happy with either of these options:

  - `/^([^]+)[/]([^]+?)(?:@(\d+))?$/`
  - `new RegExp('^([^]+)/([^]+?)(?:@([0-9]+))?$')`

In fact, since this is such an important pattern it would be nice to include comments:

```javascript
var pattern = new RegExp(
  '^'
+ '([^]+)'      //  <namespace>
+ '/'           //  SOLIDUS (U+002F)
+ '([^]+?)'     //  <name>
+ '(?:'         //  optional non-capturing group {
+   '@'         //    COMMERCIAL AT (U+0040)
+   '([0-9]+)'  //    <version>
+ ')?'          //  }
+ '$'
);
```